source/Graphite.System/AppPoolListener.cs

```diff
@@ -11,65 +11,84 @@ namespace Graphite.System
     internal class AppPoolListener
     {
         private readonly string appPoolName;
+        private readonly string category = "Process";
```

Comment:
Is there a use case for any value other than "Process"? Because we're resolving the AppPool name to a process id.